.drone.yml

```diff
@@ -38,10 +39,11 @@ pipeline:
         MODE: OLD
 
   translation-reader-make:
-    image: owncloudci/transifex:latest
+    image: ${TRANSLATION_READER_IMAGE=owncloudci/transifex:latest}
     pull: true
     secrets: [ tx_token ]
     commands:
+    - cd ${REPO_PATH}
```

Comment:
Changing the directory shouldn't be needed anymore, right?

Reply:
Yes, it is needed. 
It even has two separate use cases:
1) I'm using it for local testing, see my change to README.md. I can clone to a subfolder and pass that folder as `REPO_PATH`
2) The desktop client has two separate Makefiles, one for the actual client, one for the installer. Even if I combine them, they must not be in the root folder of the repository, as that would break in source builds with the CMake Makefile generator. So I need `REPO_PATH` to specify the subfolder.

Reply:
I dislike the directory change... The make targets had been the goal to unify the process accross the involved projects.

Reply:
It just doesn't always make sense:
1. If there is more than one translation project in a repo we might need different tools: for example qt tools and nsis tools (or whatever).
2. It might complicate the implementation of the Makefile a lot. The main Makefile of owncloud/core e.g. has checks for the availability of tools (composer, ...) which are not needed for translations. Hacking around those is certainly worse than this `cd` imho.